src/app/static/src/js/components/runWorkflow/cancelDialog.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div id="confirm-cancel-container"
+         v-bind:class="['modal-background', {'modal-hide':!showModal}, {'modal-show':showModal}]">
+        <div class="modal-main px-3 py-3">
+            <div id="cancel-heading" class="mb-2 font-weight-bold">Confirm cancellation</div>
+            <div id="cancel-text" class="mb-2">Are you sure you want to cancel ?</div>
```

Comment:
Space looks a bit odd here:
```suggestion
            <div id="cancel-text" class="mb-2">Are you sure you want to cancel?</div>
```